### DISCUSSION TITLE
Where can I find the source code of the image summerwind/actions-runner

### DISCUSSION THREAD

**Discussion by wvides** [actions/actions-runner-controller#1034] [Questions] [answered]
I've been looking for the source code of the image `summerwind/actions-runner` user by the controller but have been unable to find it, can someone point me in the right direction? thanks

Thanks

**Comment by rossrollin** [comment, marked as answer]
https://github.com/actions-runner-controller/actions-runner-controller/blob/master/runner/Dockerfile
